shell/pkg/vue.config.js

```diff
@@ -53,6 +53,7 @@ module.exports = function(dir) {
       config.resolve.alias['./node_modules'] = path.join(maindir, 'node_modules');
       config.resolve.alias['@pkg'] = dir;
       config.resolve.alias['~pkg'] = dir;
+      config.resolve.alias[`@/pkg/${ pkgName }`] = dir;
```

Comment:
This feels like a workaround, it's best to convert references from `@/pkg` to `@pkg`. There's some tech debt here as well, we (UI team) need to remove the `~pkg` alias (we have a gate that will fail if it's used)

Reply:
Thanks for your review. Should I use `@pkg/harvester/config/labels-annotations` or `@pkg/config/labels-annotations`?  If I use `@pkg/config`, the unit test will be failed and throw an error that can not locale module @pkg/config/labels-annotations

https://github.com/rancher/dashboard/blob/07683d34a651d52416ebdbd90ec97135f0975116/jest.config.js#L15

Reply:
Thank you @richard-cox! Considering rancher-component package name is `@rancher/components`, I use the directory name as `pkgName`. 

And `'@pkg/(.*)': '<rootDir>/pkg/$1'` only works for `@pkg/harvester/config` not `@pkg/config`, it requires the `pkgName` to locale the directory path.